cdk/__snapshots__/infra.test.ts.snap

```diff
@@ -1174,6 +1174,61 @@ systemctl start app
       "Type": "AWS::S3::Bucket",
       "UpdateReplacePolicy": "Retain",
     },
+    "staticPolicy36384B58": Object {
+      "Properties": Object {
+        "Bucket": Object {
+          "Ref": "staticD8C87B36",
+        },
+        "PolicyDocument": Object {
+          "Statement": Array [
+            Object {
+              "Action": "s3:PutObject",
+              "Effect": "Allow",
+              "Principal": Object {
+                "AWS": "arn:aws:iam::702972749545:role/galaxies-data-refresher-lambda-role-PROD",
```

Comment:
Could we add the `@guardian/private-infrastructure-config` mock to ensure account IDs (private data) is not in VCS?

Reply:
done! good spot, thank you @akash1810